sources/models/src/lib.rs

```diff
@@ -101,7 +101,8 @@ use crate::modeled_types::{
     DNSDomain, ECSAgentLogLevel, ECSAttributeKey, ECSAttributeValue, FriendlyVersion, Identifier,
     KubernetesAuthenticationMode, KubernetesBootstrapToken, KubernetesClusterName,
     KubernetesLabelKey, KubernetesLabelValue, KubernetesTaintValue,
-    Lockdown, SingleLineString, SysctlKey, Url, ValidBase64,
+    Lockdown, SingleLineString, SysctlKey, Url, ValidBase64, KubernetesEvictionHardKey, KubernetesThresholdValue,
+    KubernetesReservedResourceKey, KubernetesQuantityValue,
```

Comment:
super-nit: put these `Kubernetes` imports up with the other ones, optionally in alphabetical order